pages.fr/common/deno.md

```diff
@@ -0,0 +1,24 @@
+# deno
+
+> Un environnement d’exécution sécurisé pour JavaScript et TypeScript.
+> Plus d'information : <https://deno.land>.
+
+- Exécuter un fichier JavaScript ou TypeScript :
```

Comment:
Command and example descriptions on pages in French must use the third person singular present indicative tense, cf. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#french-specific-rules